go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/mattermost/mattermost-server/v5 v5.18.0
 	github.com/pkg/errors v0.8.1
 	github.com/stretchr/testify v1.4.0
+	github.com/vektra/mockery v0.0.0-20181123154057-e78b021dcbb5 // indirect
```

Comment:
I can change to go-mock no problem. I picked mockery because I saw it was used in [Mattermost-server](https://github.com/mattermost/mattermost-server/blob/7035e09fe9bbfd981bf73990d7857535f7b503aa/Makefile#L211). Can you expand on when go-mock is preferred?

Reply:
I have changed to go-mock